docs/scripting/trust_scripts/corps.top.mdx

```diff
@@ -155,6 +155,6 @@ Return an object.
 ```js
 function (context, args)
 {
-  return #ns.corps.top()
+  return #fs.corps.top()
```

Comment:
`corps.top` is NULLSEC
![image](https://github.com/comcode-org/hackmud_wiki/assets/24510197/14213792-2446-44df-bbbd-d77e50f75753)

Reply:
Good catch, fixed 8858d667c0c9879b171a02f989d7d926aac34827